client/src/pages/ExistingProgramme/editRequirementsBox.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import {
+    Flex,
+    Text,
+    Box,
+    Select,
+    NumberInput,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper,
+    Stack,
+} from "@chakra-ui/core";
+import { AiOutlineCheckCircle, AiOutlineCloseCircle } from "react-icons/ai";
+
+const EditRequirementsBox = ({ closeEdit }) => {
+    return (
+        <Flex
+            alignItems="center"
+            alignSelf="center"
+            direction="column"
+            textAlign="flex-start"
+            marginTop="20px"
+            height="150px"
+            width="85%"
+            color="white"
+            bg="#e2e2e2"
+        >
+            <Text textAlign="center" fontSize="md" color="#000000" as="i">
```

Comment:
Extra padding at the top again would be great :D

Reply:
done